### PR TITLE
[Gekidou] Add color to loading spinners

### PR DESCRIPTION
**Pull request by larkox**
#### Summary
Add color to loading spinners. This is based on our discussions on the channel and my PR on browse channels.

This small PR will allow @avinashlng1080 to work with this changes, and reduce the size of the browse channels PR.

#### Ticket Link
None

#### Release Note
```release-note
None
```

### CODE REVIEW

**Review by enahum — APPROVED** [review]

**Comment by enahum** [comment]
@avinashlng1080 please review asap

File: app/screens/channel/failed_teams/index.tsx [hunk 1]
```diff
@@ -27,7 +29,16 @@ const FailedTeams = () => {
     }, []);
 
     if (loading) {
-        return <Loading/>;
+        return (
+            <Loading
+                containerStyle={{
+                    flex: 1,
```
Comment (avinashlng1080) [inline app/screens/channel/failed_teams/index.tsx:35]: Should this be in Stylesheet ?

**Review by avinashlng1080 — CHANGES_REQUESTED** [review]
LGTM, just one comment regarding the containerStyle.

**Review by avinashlng1080 — APPROVED** [review]
LGTM

**Pull request merged**